.craft.yml

```diff
@@ -0,0 +1,61 @@
+minVersion: '0.30.1'
+changelogPolicy: auto
+preReleaseCommand: >-
+  node -p "
+    const { execSync } = require('child_process');
+    const packages = Object.values(
+      JSON.parse(execSync('yarn -s workspaces info', { env: { FORCE_COLOR: 0 } }))
+    ).map(({ location }) => location);
+    packages
+      .map(package =>
+        execSync(
+          [
+            'yarn',
+            '--no-git-tag-version',
+            'version',
+            '--new-version',
+            process.env.CRAFT_NEW_VERSION,
+          ].join(' '),
+          { cwd: package }
+        ).toString()
+      )
+      .join('\n');
```

Comment:
Do we need to map packages (e.g. as opposed to `forEach`) if they don't get returned?

Reply:
This is to get the output as `node -p` prints the result of the last expression and in `trace` logging level, Craft emits these for debugging.